pkg/codegen/testing/test/testdata/plain-object-defaults/nodejs/types/mod2/index.ts

```diff
@@ -0,0 +1,7 @@
+// *** WARNING: this file was generated by test. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+export * from "./input";
+export * from "./output";
+export * as inputs from "./input";
+export * as outputs from "./output";
```

Comment:
@t0yv0 This is the only major change of note since your last review.

Each file matching the format `types/{*}/index.ts` will export names two ways. I'll explain.

Suppose you have a module `/types/foo/index.ts`, and an input named `MyInput`.

1. You can import `MyInput` on the index itself: 

```typescript
import * as foo from "types/foo";
const input: MyInput = null;
```

2. You can also import it from inside the `input` namespace on the index:

```typescript
import * as foo from "types/foo";
const input: foo.input.MyInput = null;
import * as fooInputs from "types/foo/input";
const input2: fooInputs.MyInput = null;
```
It's kind of gross to expose the name twice. However, as we know from #10442 there are several locations where imports aren't smart enough to identify the exact name of the required import. I'm aiming to fix that when I tackle #10442.

Reply:
Nevermind this comment. I removed the second namespace, since it's not required for backwards compatibility and makes the API really confusing.